kotlin/src/main/com/looker/sdk/4.0/methods.kt

```diff
@@ -3207,11 +3011,10 @@ class LookerSDK(authSession: AuthSession) : APIMethods(authSession) {
      */
     fun create_content_favorite(
         body: WriteContentFavorite
-    ) : SDKResponse {
+    ): SDKResponse {
```

Comment:
This all LGTM but want to get @drstrangelooker's  opinion on formating the autogen. 

When the SDK is regenerated for the next Looker release we'll probably need to rerun the Kotlin formatter. Is it possible to make that part of the `gen` task?

Reply:
I have no idea about formatting. Sorry